src/main/kotlin/com/github/ashleybudtke/happyplants/domain/Water.kt

```diff
@@ -0,0 +1,12 @@
+package com.github.ashleybudtke.happyplants.domain
+
+import com.fasterxml.jackson.annotation.JsonFormat
+
+@JsonFormat(shape = JsonFormat.Shape.OBJECT)
```

Comment:
Didn't know about this annotation, pretty sweet! In my reading, it looks like it won't properly deserialize JSON back into this enum unfortunately 😞 (see javadocs https://github.com/FasterXML/jackson-annotations/blob/master/src/main/java/com/fasterxml/jackson/annotation/JsonFormat.java). This probably hasn't come up because you're currently just GETing the plants. FYI.